.github/workflows/test-energyflow.yml

```diff
@@ -20,6 +20,7 @@ jobs:
   build:
     runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
```

Comment:
This makes sense, but if you can leave a comment noting this is for Windows that might be useful later.